install/gcp/upgrade_tools/db_migrator.py

```diff
@@ -0,0 +1,118 @@
+import sys
+
+# Importing migrate.changeset adds some new methods to existing SQLAlchemy
+# objects but we will not be calling the library directly.
+import migrate.changeset  # noqa: F401, pylint: disable=unused-import
+from sqlalchemy.exc import OperationalError
+
+import google.cloud.forseti.services.scanner.dao as scanner_dao
+import google.cloud.forseti.services.inventory.storage as inventory_dao
+import google.cloud.forseti.services.dao as general_dao
+
+from google.cloud.forseti.common.util import logger
+
+
+DEFAULT_DB_CONN_STR = 'mysql://root@127.0.0.1:3306/forseti_security'
+LOGGER = logger.get_logger(__name__)
+
+
+class ColumnAction:
+    """Column action class."""
+    DROP = 'DROP'
+    CREATE = 'CREATE'
+
+
+def create_column(table, column):
+    """Create Column.
+
+    Args:
+        table (sqlalchemy.schema.Table): The sql alchemy table object.
+        column (sqlalchemy.schema.Column): The sql alchemy column object.
+    """
+    LOGGER.info('Attempting to create column: %s', column.name)
+    column.create(table, populate_default=True)
+
+
+def drop_column(table, column):
+    """Create Column.
+
+    Args:
+        table (sqlalchemy.schema.Table): The sql alchemy table object.
+        column (sqlalchemy.schema.Column): The sql alchemy column object.
+    """
+    LOGGER.info('Attempting to drop column: %s', column.name)
+    column.drop(table)
+
+
+column_action_mapping = {ColumnAction.DROP: drop_column,
+                         ColumnAction.CREATE: create_column}
+
+
+def migrate_schema(engine, base):
+    """Create all tables in the database if not existing.
+
+    Args:
+        engine (object): Database engine to operate on.
+        base (Base): Declarative base.
+    """
+    # Create tables if not exists.
+    base.metadata.create_all(engine)
+    base.metadata.bind = engine
+
+    # Update schema changes.
+    # Find all the child classes inherited from declarative base class.
+    base_subclasses = _find_subclasses(base)
+
+    # Find all the Table objects for each of the classes.
+    # The format of tables is: {table_name: Table object}.
+    tables = base.metadata.tables
+
+    schema_update_actions_method = 'get_schema_update_actions'
+
+    for subclass in base_subclasses:
+        update_actions = getattr(subclass, schema_update_actions_method, None)
```

Comment:
This line here needs to be consistent with the naming comment above.

Reply:
Updated